server/src/model/city_configuration.py

```diff
@@ -1,8 +1,18 @@
-from pydantic import BaseModel
+from typing import List
+
+from pydantic import BaseModel, Field
+
+
+class TramStopPairCheck(BaseModel):
+    from_: int = Field(alias="from")
+    to: int
+    ratio: float
 
 
 class CityConfiguration(BaseModel):
     osm_area_name: str
     gtfs_url: str
     ignored_gtfs_lines: list[str]
     custom_stop_mapping: dict[str, int]
+    max_distance_ratio: float
+    custom_tram_stop_pair_max_distance_checks: List[TramStopPairCheck]
```

Comment:
Importing `List` from `typing` is deprecated since Python 3.9 I believe:
```suggestion
    custom_tram_stop_pair_max_distance_checks: list[TramStopPairCheck]
```

Reply:
Changed to `list`